core/src/main/scripts/importer/validateData.py

```diff
@@ -1346,6 +1348,30 @@ def checkSwissProt(self, value):
                 return True
         # if no reasons to return with a message were found, return valid
         return True
+    
+    def checkStartPosition(self, value):
+        """Check that the Start_Position value is an integer."""
+        if value.isdigit() == False or (value.isdigit() and '.' in value):
+            self.logger.error(
+                'The start position of this gene is not '
```

Comment:
According to [the MAF specification](https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+(MAF)+Specification):

> Lowest numeric position of the reported variant on the genomic reference sequence. Mutation start coordinate (1-based coordinate system).